src/handlers.ts

```diff
@@ -35,21 +36,23 @@ export function getBuildImage(req: Request, res: Response, next: NextFunction):
         .catch(next);
 }
 
-export function enqueueBuild(req: Request, res: Response, next: NextFunction): void {
-    Promise.resolve()
-        .then(() => assertIdPathParam(req))
-        .catch(next);
+export async function enqueueBuild(req: Request, res: Response, next: NextFunction): Promise<void> {
+    const { originalname } = await req.file;
+    winston.info(`Team ID: ${req.params.teamId}`);
+    winston.info(`File Name: ${originalname}`);
+    res.json({ originalname });
+    res.end();
 }
 
 /**
  * Assert that if the request contains the `id` path parameter, it is a string. Throw 400 otherwise.
  * @param req 
  */
 function assertIdPathParam(req: Request) {
-    if(_.isNil(req.params.id)) {
+    if (_.isNil(req.params.id)) {
```

Comment:
This should be added as a tslint rule before we start making arbitrary stylistic changes